var/spack/repos/builtin/packages/nek5000/package.py

```diff
@@ -88,17 +88,14 @@ def install(self, spec, prefix):
         toolsDir   = 'tools'
         binDir     = 'bin'
 
-        F77 = self.compiler.fc
+        FC  = self.compiler.fc
         CC  = self.compiler.cc
 
         # Build the tools, maketools copy them to Nek5000/bin by default.
         # We will then install Nek5000/bin under prefix after that.
         with working_dir(toolsDir):
             # Update the maketools script to use correct compilers
-            if self.version == Version('17.0.0-beta2'):  # Old release
-                filter_file(r'^F77\s*=.*', 'F77=\"' + F77 + '\"', 'maketools')
-            else:
-                filter_file(r'^FC\s*=.*', 'FC=\"' + F77 + '\"', 'maketools')
+            filter_file(r'^FC\s*=.*', 'FC=\"' + FC + '\"', 'maketools')
             filter_file(r'^CC\s*=.*', 'CC=\"' + CC  + '\"',   'maketools')
```

Comment:
You don't need to escape these `'"'`. Same below. Personally, I would use `'CC="{0}"'.format(CC)'`, but this is fine too.

Reply:
Thanks Adam !! I will fix this.